x/programs/rust/wasmlanche_sdk/src/types.rs

```diff
@@ -81,8 +81,12 @@ impl From<i64> for Bytes32 {
     }
 }
 
+/// A trait that represents an argument that can be passed to & from the host.
 pub trait Argument {
     fn as_bytes(&self) -> Cow<'_, [u8]>;
+    fn from_bytes(bytes: &[u8]) -> Self
+    where
+        Self: Sized;
```

Comment:
I think you might as well put the bound on the trait, 
```rust
pub trait Argument: Sized {
```

Reply:
deleted `from_bytes` for argument so gonna resolve a lot of these comments